examples/simple-dg.py

```diff
@@ -233,8 +239,8 @@ def get_inverse_mass_matrix(self, grp, dtype):
         return actx.freeze(actx.from_numpy(matrix))
 
     def inverse_mass(self, vec):
-        if isinstance(vec, np.ndarray):
-            return obj_array_vectorize(self.inverse_mass, vec)
+        if not isinstance(vec, DOFArray):
+            return map_dof_array_container(self.inverse_mass, vec)
```

Comment:
```suggestion
        if not isinstance(vec, DOFArray):
            return array_container_vectorize(self.inverse_mass, vec)
```
? This makes more sense to me because the function is recursive by itself (so using the recursive `map_dof_array_container`) is weird.

Reply:
Maybe we should rename these two to `rec_map_array_container` (instead of `map_array_container`) and `map_array_container` (instead of `array_container_vectorize`) to make it clear which is recursive?

Reply:
I like your suggestion of naming them `map` and `map_rec`.

As for this thing, I did that on purpose, and it didn't feel particularly weird to me. Kind of like, "call me once you've got a `DOFArray`".